package.json

```
@@ -34,12 +34,12 @@
   },
   "browserslist": "Node 10.17 - 11 and Node < 11, Node 12 - 13 and Node < 13, Node >= 13.7, > 0.5%, not OperaMini all, not dead",
   "peerDependencies": {
-    "graphql": "^0.11.3 || ^0.12.3 || ^0.13.0 || ^14.0.0 || ^15.0.0"
+    "graphql": "14 - 15",
+    "subscriptions-transport-ws": "^0.9.0"
```

https://github.com/apollographql/apollo-client/blob/8b8e9d0822a3268723621b107165041832d732a0/package.json#L57-L61

```suggestion
    "graphql": ">=14.0.0",
    "subscriptions-transport-ws": ">=0.9.0"
```